pkgs/ffigen/test/code_generator_tests/expected_bindings/_expected_global_native_bindings.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2023, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// AUTO GENERATED FILE, DO NOT EDIT.
+//
+// Generated by `package:ffigen`.
+// ignore_for_file: type=lint
+import 'dart:ffi' as ffi;
+
+@ffi.Native<ffi.Int32>()
+external int test1;
+@ffi.Native<ffi.Pointer<ffi.Float>>()
```

Comment:
Add new lines between the generated declarations.

Why don't we have new lines here, but we do in pkgs/ffigen/test/code_generator_tests/expected_bindings/_expected_native_symbol_bindings.dart ?

Reply:
Native functions (and all other bindings) add another newline after their definition, native globals didn't. Fixed!